<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPSeverClass</class>
 <widget class="QWidget" name="TCPSeverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCPSever</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTextBrowser" name="textRecv"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="textSend">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>192</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>发送</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonExit">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>191</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tcpsever.qrc"/>
 </resources>
 <connections/>
</ui>
